server/libs/domain/src/media/media.service.ts

```diff
@@ -257,7 +257,7 @@ export class MediaService {
 
     try {
       const faceId = randomUUID();
```

Comment:
In general, we've been using the crypto repository for stuff like this, so that it can be mocked in tests.